ccm_web/server/src/config.ts

```diff
@@ -30,9 +30,19 @@ interface CanvasConfig {
   instanceURL: string
   apiClientId: string
   apiSecret: string
+  adminApiToken: string
+  newUserAccountID: number
   customCanvasRoleData: CustomCanvasRoleData
 }
 
+export interface InvitationConfig {
+  apiURL: string
```

Comment:
This is causing a compilation error, I think you just forgot to remove it when you made it a constant.

Reply:
;finished;